packages/frontend/tests/integration/components/learner-groups/loading-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { setupIntl } from 'ember-intl/test-support';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+
+module('Integration | Component | learner-groups/loading', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks, 'en-us');
+
+  test('it renders', async function (assert) {
+    await render(hbs`<LearnerGroups::Loading />`);
```

Comment:
Should test the `count` even though we're not using it quite yet. Hopefully we'll get it working at some point and then we should have a test for it.

Reply:
There is a test for this now.